src/aiida/manage/tests/pytest_fixtures.py

```diff
@@ -504,44 +504,49 @@ def get_code(entry_point, executable, computer=aiida_localhost, label=None, **kw
 
 @pytest.fixture(scope='session')
 def ssh_key(tmp_path_factory) -> t.Generator[pathlib.Path, None, None]:
-    """Generate a temporary SSH key pair for the test session and return the filepath of the private key.
+    """Returns a SSH key for the test session. If the environment variable ``AIIDA_PYTEST_SSH_KEY`` is set we take the
+    key from this path otherwise we generate a temporary SSH key pair for the test session and return the filepath of
+    the private key.
```

Comment:
We try and keep the PEP convention of having docstring title be a single line. If there is more, an empty line is inserted.
```suggestion
    """Returns a SSH key for the test session.

    If the environment variable ``AIIDA_PYTEST_SSH_KEY`` is set we take the key from this path otherwise we generate a
    temporary SSH key pair for the test session and return the filepath of the private key.
```